### DISCUSSION TITLE
How to build an application for linux?

### DISCUSSION THREAD

**Discussion by YoriMirus** [AvaloniaUI/Avalonia#8002] [Q&A]
Hello, I have an Avalonia application I would like to try running on linux. I tried looking on the internet but couldn't find anything about it, despite the fact that Avalonia is cross-platform.

So what do I need to do? Is building enough or do I need to specify something so that my application can be opened on Linux machines?

**Comment by sn4k3** [comment]
You need to `dotnet publish` for linux so users dont need to install dotnet runtime
Then just distribute via zip or AppImage

My build script: https://github.com/sn4k3/UVtools/blob/master/build/createRelease.sh
